[2024-05-22 14:01:14,015: INFO: main: Welcome]
[2024-05-22 20:16:30,222: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-22 20:16:30,222: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 20:16:30,222: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 20:16:30,222: INFO: common: created directory at: artifacts]
[2024-05-22 20:16:30,222: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-22 20:16:30,230: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 20:17:01,233: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 20:17:01,679: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-22 20:59:49,220: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 20:59:49,311: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 20:59:49,317: INFO: common: created directory at: artifacts]
[2024-05-22 20:59:49,317: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 20:59:57,649: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 20:59:57,655: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 20:59:57,658: INFO: common: created directory at: artifacts]
[2024-05-22 20:59:57,658: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 21:00:07,339: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 21:00:07,339: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 21:00:07,345: INFO: common: created directory at: artifacts]
[2024-05-22 21:00:07,345: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 21:02:05,347: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 21:02:05,358: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 21:02:05,358: INFO: common: created directory at: artifacts]
[2024-05-22 21:02:05,360: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 21:02:05,748: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 21:02:05,982: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 21:28:22,520: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-22 21:28:22,521: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 21:28:22,526: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 21:28:22,529: INFO: common: created directory at: artifacts]
[2024-05-22 21:28:22,530: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-22 21:28:22,530: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 21:28:56,148: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 21:28:56,699: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-22 21:28:56,700: INFO: main: *******************]
[2024-05-22 21:28:56,700: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-22 21:28:56,700: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 21:28:56,701: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 21:28:56,703: INFO: common: created directory at: artifacts]
[2024-05-22 21:28:56,703: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 21:28:57,217: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 21:28:57,475: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 21:28:58,173: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-22 21:59:04,748: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 21:59:04,836: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 21:59:04,836: INFO: common: created directory at: artifacts]
[2024-05-22 21:59:04,836: INFO: common: created directory at: artifacts\training]
[2024-05-22 21:59:06,682: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-22 22:22:09,943: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-22 22:22:09,945: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:22:09,950: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:22:09,952: INFO: common: created directory at: artifacts]
[2024-05-22 22:22:09,953: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-22 22:22:09,953: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 22:22:42,686: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 22:22:43,173: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-22 22:22:43,173: INFO: main: *******************]
[2024-05-22 22:22:43,173: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-22 22:22:43,173: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:22:43,173: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:22:43,173: INFO: common: created directory at: artifacts]
[2024-05-22 22:22:43,173: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 22:22:43,482: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 22:22:43,623: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 22:22:43,720: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-22 22:22:43,720: INFO: main: *******************]
[2024-05-22 22:22:43,720: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-22 22:22:43,724: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:22:43,724: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:22:43,724: INFO: common: created directory at: artifacts]
[2024-05-22 22:22:43,724: INFO: common: created directory at: artifacts\training]
[2024-05-22 22:22:43,865: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-22 22:22:43,865: ERROR: main: [WinError 3] The system cannot find the path specified: 'artifacts\\data_ingestion\\kidney-ct-scan-image']
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 20, in main
    training.train_valid_generator()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 39, in train_valid_generator
    self.valid_generator = valid_datagenerator.flow_from_directory(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\legacy\preprocessing\image.py", line 1138, in flow_from_directory
    return DirectoryIterator(
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\legacy\preprocessing\image.py", line 453, in __init__
    for subdir in sorted(os.listdir(directory)):
                         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'artifacts\\data_ingestion\\kidney-ct-scan-image'
[2024-05-22 22:27:55,341: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-22 22:27:55,341: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:27:55,341: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:27:55,341: INFO: common: created directory at: artifacts]
[2024-05-22 22:27:55,341: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-22 22:27:55,341: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 22:28:26,582: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 22:28:27,091: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-22 22:28:27,091: INFO: main: *******************]
[2024-05-22 22:28:27,091: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-22 22:28:27,091: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:28:27,091: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:28:27,091: INFO: common: created directory at: artifacts]
[2024-05-22 22:28:27,091: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 22:28:27,381: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 22:28:27,540: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 22:28:27,623: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-22 22:28:27,623: INFO: main: *******************]
[2024-05-22 22:28:27,623: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-22 22:28:27,623: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:28:27,632: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:28:27,632: INFO: common: created directory at: artifacts]
[2024-05-22 22:28:27,632: INFO: common: created directory at: artifacts\training]
[2024-05-22 22:28:27,776: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-22 22:28:28,893: ERROR: main: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.]
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 21, in main
    training.train()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 78, in train
    self.model.fit(
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\utils\traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\optimizers\base_optimizer.py", line 228, in _check_variables_are_known
    raise ValueError(
ValueError: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.
[2024-05-22 22:34:09,361: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-22 22:34:09,361: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:34:09,361: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:34:09,361: INFO: common: created directory at: artifacts]
[2024-05-22 22:34:09,361: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-22 22:34:09,377: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 22:34:41,764: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-22 22:34:42,240: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-22 22:34:42,240: INFO: main: *******************]
[2024-05-22 22:34:42,240: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-22 22:34:42,256: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:34:42,256: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:34:42,256: INFO: common: created directory at: artifacts]
[2024-05-22 22:34:42,256: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-22 22:34:42,541: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 22:34:42,692: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-22 22:34:42,772: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-22 22:34:42,772: INFO: main: *******************]
[2024-05-22 22:34:42,772: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-22 22:34:42,772: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-22 22:34:42,772: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-22 22:34:42,786: INFO: common: created directory at: artifacts]
[2024-05-22 22:34:42,786: INFO: common: created directory at: artifacts\training]
[2024-05-22 22:34:42,956: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-22 22:34:43,672: ERROR: main: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.]
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 21, in main
    training.train()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 78, in train
    self.model.fit(
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\utils\traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\optimizers\base_optimizer.py", line 228, in _check_variables_are_known
    raise ValueError(
ValueError: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.
[2024-05-23 01:52:35,903: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 01:52:35,911: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 01:52:35,914: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 01:52:35,914: INFO: common: created directory at: artifacts]
[2024-05-23 01:52:35,915: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 01:52:35,915: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 01:53:06,342: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 01:53:07,086: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 01:53:07,087: INFO: main: *******************]
[2024-05-23 01:53:07,087: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 01:53:07,089: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 01:53:07,090: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 01:53:07,091: INFO: common: created directory at: artifacts]
[2024-05-23 01:53:07,091: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 01:53:07,461: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 01:53:08,479: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 01:53:09,115: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 01:53:09,116: INFO: main: *******************]
[2024-05-23 01:53:09,116: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 01:53:09,118: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 01:53:09,120: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 01:53:09,120: INFO: common: created directory at: artifacts]
[2024-05-23 01:53:09,121: INFO: common: created directory at: artifacts\training]
[2024-05-23 01:53:09,289: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 01:53:10,343: ERROR: main: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.]
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 21, in main
    training.train()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 72, in train
    self.model.fit(
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\utils\traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\optimizers\base_optimizer.py", line 228, in _check_variables_are_known
    raise ValueError(
ValueError: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.
[2024-05-23 02:33:13,297: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 02:33:13,328: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 02:33:13,332: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 02:33:13,333: INFO: common: created directory at: artifacts]
[2024-05-23 02:33:13,333: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 02:33:13,333: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 02:33:45,616: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 02:33:45,960: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 02:33:45,961: INFO: main: *******************]
[2024-05-23 02:33:45,961: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 02:33:45,963: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 02:33:45,963: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 02:33:45,964: INFO: common: created directory at: artifacts]
[2024-05-23 02:33:45,964: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 02:33:46,320: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 02:33:46,670: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 02:33:46,843: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 02:33:46,844: INFO: main: *******************]
[2024-05-23 02:33:46,844: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 02:33:46,846: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 02:33:46,846: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 02:33:46,847: INFO: common: created directory at: artifacts]
[2024-05-23 02:33:46,847: INFO: common: created directory at: artifacts\training]
[2024-05-23 02:33:47,015: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 02:34:18,811: ERROR: main: 'NoneType' object has no attribute 'items']
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 21, in main
    training.train()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 78, in train
    self.model.fit(
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\utils\traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\backend\tensorflow\trainer.py", line 350, in fit
    "val_" + name: val for name, val in val_logs.items()
                                        ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'items'
[2024-05-23 03:06:08,506: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 03:06:08,596: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 03:06:08,596: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 03:06:08,596: INFO: common: created directory at: artifacts]
[2024-05-23 03:06:08,596: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 03:06:08,596: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 03:06:40,650: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 03:06:40,978: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 03:06:40,978: INFO: main: *******************]
[2024-05-23 03:06:40,978: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 03:06:40,978: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 03:06:40,994: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 03:06:40,994: INFO: common: created directory at: artifacts]
[2024-05-23 03:06:40,994: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 03:06:41,350: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 03:06:41,633: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 03:06:42,004: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 03:06:42,004: INFO: main: *******************]
[2024-05-23 03:06:42,004: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 03:06:42,004: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 03:06:42,004: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 03:06:42,004: INFO: common: created directory at: artifacts]
[2024-05-23 03:06:42,004: INFO: common: created directory at: artifacts\training]
[2024-05-23 03:06:42,164: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 03:06:42,164: ERROR: main: ImageDataGenerator.flow_from_directory() got an unexpected keyword argument 'workers']
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 20, in main
    training.train_valid_generator()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 44, in train_valid_generator
    self.valid_generator = valid_datagenerator.flow_from_directory(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: ImageDataGenerator.flow_from_directory() got an unexpected keyword argument 'workers'
[2024-05-23 03:31:46,438: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 03:31:46,438: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 03:31:46,438: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 03:31:46,438: INFO: common: created directory at: artifacts]
[2024-05-23 03:31:46,438: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 03:31:46,438: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 03:32:18,261: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 03:32:18,587: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 03:32:18,587: INFO: main: *******************]
[2024-05-23 03:32:18,587: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 03:32:18,587: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 03:32:18,587: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 03:32:18,587: INFO: common: created directory at: artifacts]
[2024-05-23 03:32:18,587: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 03:32:18,902: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 03:32:19,061: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 03:32:20,143: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 03:32:20,143: INFO: main: *******************]
[2024-05-23 03:32:20,143: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 03:32:20,159: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 03:32:20,159: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 03:32:20,159: INFO: common: created directory at: artifacts]
[2024-05-23 03:32:20,159: INFO: common: created directory at: artifacts\training]
[2024-05-23 03:32:20,159: ERROR: main: 'Training' object has no attribute 'get_base_model']
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 19, in main
    training.get_base_model()
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Training' object has no attribute 'get_base_model'
[2024-05-23 11:32:18,162: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 11:32:18,183: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 11:32:18,192: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 11:32:18,192: INFO: common: created directory at: artifacts]
[2024-05-23 11:32:18,192: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 11:32:18,192: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 11:32:51,505: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 11:32:51,872: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 11:32:51,872: INFO: main: *******************]
[2024-05-23 11:32:51,872: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 11:32:51,872: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 11:32:51,872: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 11:32:51,872: INFO: common: created directory at: artifacts]
[2024-05-23 11:32:51,880: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 11:32:52,240: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 11:32:52,423: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 11:32:52,542: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 11:32:52,542: INFO: main: *******************]
[2024-05-23 11:32:52,542: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 11:32:52,542: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 11:32:52,542: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 11:32:52,552: INFO: common: created directory at: artifacts]
[2024-05-23 11:32:52,552: INFO: common: created directory at: artifacts\training]
[2024-05-23 11:32:52,752: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 11:32:53,725: ERROR: main: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.]
Traceback (most recent call last):
  File "F:\kidney_diseases_classification\main.py", line 35, in <module>
    model_trainer.main()
  File "F:\kidney_diseases_classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 21, in main
    training.train()
  File "F:\kidney_diseases_classification\src\cnnClassifier\components\model_training.py", line 74, in train
    self.model.fit(
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\utils\traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\Mallik\AppData\Roaming\Python\Python311\site-packages\keras\src\optimizers\base_optimizer.py", line 228, in _check_variables_are_known
    raise ValueError(
ValueError: Unknown variable: <KerasVariable shape=(25088, 4), dtype=float32, path=dense/kernel>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.
[2024-05-23 11:39:55,428: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 11:39:55,428: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 11:39:55,432: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 11:39:55,432: INFO: common: created directory at: artifacts]
[2024-05-23 11:39:55,432: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 11:39:55,432: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 11:40:26,261: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 11:40:26,657: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 11:40:26,657: INFO: main: *******************]
[2024-05-23 11:40:26,664: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 11:40:26,664: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 11:40:26,664: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 11:40:26,664: INFO: common: created directory at: artifacts]
[2024-05-23 11:40:26,664: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 11:40:26,993: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 11:40:27,140: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 11:40:27,228: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 11:40:27,228: INFO: main: *******************]
[2024-05-23 11:40:27,228: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 11:40:27,228: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 11:40:27,228: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 11:40:27,228: INFO: common: created directory at: artifacts]
[2024-05-23 11:40:27,228: INFO: common: created directory at: artifacts\training]
[2024-05-23 11:40:27,389: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 11:41:03,400: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 11:41:03,508: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-05-23 17:25:14,985: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 17:25:15,085: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 17:25:15,087: INFO: common: created directory at: artifacts]
[2024-05-23 17:25:15,360: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 17:25:49,646: INFO: common: json file saved at: scores.json]
[2024-05-23 17:39:12,946: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 17:39:12,956: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 17:39:12,958: INFO: common: created directory at: artifacts]
[2024-05-23 17:39:13,190: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 17:39:45,202: INFO: common: json file saved at: scores.json]
[2024-05-23 17:42:57,181: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=5, redirect=5, status=5)) after connection broken by 'SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2427)')': /ashmitaroy17/kidney_diseases_classification.mlflow/api/2.0/mlflow-artifacts/artifacts/5f6a426b97974f36b58bcbc07c4a5f58/20f6d59c059f4175b1abfffaa98237a3/artifacts/model/data/model.keras]
[2024-05-23 19:15:32,460: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 19:15:32,460: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:15:32,468: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:15:32,468: INFO: common: created directory at: artifacts]
[2024-05-23 19:15:32,468: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 19:15:32,468: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 19:16:03,222: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 19:16:03,674: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 19:16:03,674: INFO: main: *******************]
[2024-05-23 19:16:03,674: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 19:16:03,674: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:16:03,674: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:16:03,674: INFO: common: created directory at: artifacts]
[2024-05-23 19:16:03,677: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 19:16:03,984: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 19:16:04,373: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 19:16:05,317: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 19:16:05,317: INFO: main: *******************]
[2024-05-23 19:16:05,317: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 19:16:05,317: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:16:05,317: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:16:05,317: INFO: common: created directory at: artifacts]
[2024-05-23 19:16:05,325: INFO: common: created directory at: artifacts\training]
[2024-05-23 19:16:05,477: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 19:16:40,846: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 19:16:41,057: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-05-23 19:16:41,057: INFO: main: *******************]
[2024-05-23 19:16:41,057: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-05-23 19:16:41,057: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:16:41,057: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:16:41,057: INFO: common: created directory at: artifacts]
[2024-05-23 19:16:41,232: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 19:17:17,036: INFO: common: json file saved at: scores.json]
[2024-05-23 19:17:17,036: INFO: common: json file saved at: scores.json]
[2024-05-23 19:17:17,036: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-05-23 19:26:51,026: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:26:51,031: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:26:51,034: INFO: common: created directory at: artifacts]
[2024-05-23 19:26:51,257: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 19:27:30,828: INFO: common: json file saved at: scores.json]
[2024-05-23 19:30:00,984: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-23 19:30:00,989: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:30:00,989: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:30:00,989: INFO: common: created directory at: artifacts]
[2024-05-23 19:30:00,989: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 19:30:00,989: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 19:30:30,573: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1yXkOfYmGqGMB1CvmD5qouaKV21f7jHVS/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-23 19:30:31,519: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-23 19:30:31,519: INFO: main: *******************]
[2024-05-23 19:30:31,519: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-23 19:30:31,519: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:30:31,523: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:30:31,523: INFO: common: created directory at: artifacts]
[2024-05-23 19:30:31,523: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-23 19:30:31,892: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 19:30:32,742: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 19:30:33,338: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-05-23 19:30:33,341: INFO: main: *******************]
[2024-05-23 19:30:33,341: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-05-23 19:30:33,342: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:30:33,343: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:30:33,344: INFO: common: created directory at: artifacts]
[2024-05-23 19:30:33,344: INFO: common: created directory at: artifacts\training]
[2024-05-23 19:30:33,505: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 19:31:19,912: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-05-23 19:31:21,026: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-05-23 19:31:21,026: INFO: main: *******************]
[2024-05-23 19:31:21,033: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-05-23 19:31:21,033: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:31:21,033: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:31:21,033: INFO: common: created directory at: artifacts]
[2024-05-23 19:31:21,263: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 19:32:06,889: INFO: common: json file saved at: scores.json]
[2024-05-23 19:32:06,889: INFO: common: json file saved at: scores.json]
[2024-05-23 19:32:06,889: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-05-23 19:58:40,168: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 19:58:40,172: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 19:58:40,173: INFO: common: created directory at: artifacts]
[2024-05-23 19:58:40,368: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 19:59:14,450: INFO: common: json file saved at: scores.json]
[2024-05-23 20:10:00,319: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 20:10:00,323: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 20:10:00,323: INFO: common: created directory at: artifacts]
[2024-05-23 20:10:00,507: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 20:10:29,795: INFO: common: json file saved at: scores.json]
[2024-05-23 20:14:06,679: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=5, redirect=5, status=5)) after connection broken by 'SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2427)')': /ashmitaroy17/kidney_diseases_classification.mlflow/api/2.0/mlflow-artifacts/artifacts/5f6a426b97974f36b58bcbc07c4a5f58/64233fcb46d94221bb30c1fe790de42a/artifacts/model/data/model.keras]
[2024-05-23 20:36:17,264: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 20:36:17,276: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 20:36:17,277: INFO: common: created directory at: artifacts]
[2024-05-23 20:36:17,509: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 20:36:49,127: INFO: common: json file saved at: scores.json]
[2024-05-23 22:21:12,831: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-23 22:21:12,923: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-23 22:21:12,927: INFO: common: created directory at: artifacts]
[2024-05-23 22:21:13,316: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-23 22:21:55,684: INFO: common: json file saved at: scores.json]
